Assets/Scripts/Player/PlayerStats.cs

```diff
@@ -50,23 +49,18 @@ public override void Awake()
 		}
 	}
 
-	public void IncrementMaxStat( Stat stat )
+	public void SetMaxStat( Stat stat, float maxValue )
 	{
-		StatObject statObject = _statDictionary[stat];
-		statObject.currentMax += statObject.maxIncrement;
-		ScaleCurrImage( statObject );
-	}
+		DebugUtils.Assert( maxValue > 0.0f, "Max stat value must be greater than 0." );
 
-	public void DecrementMaxStat( Stat stat )
-	{
 		StatObject statObject = _statDictionary[stat];
-		statObject.currentMax = Mathf.Max(statObject.currentMax - statObject.maxIncrement, 0.0f); // decrement and clamp at a minimum of 0
+		statObject.currentMax = maxValue; // assume maxValue is a sane value.
```

Comment:
I don't think you really need this comment here. The assert should be pretty explanatory